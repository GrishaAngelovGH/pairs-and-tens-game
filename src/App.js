function App() {
  return (
    <div className="container-fluid">
      <div className="row vh-100 bg-primary-subtle">
        <div className="col-md-12">
          <h1>App</h1>
        </div>
      </div>
    </div>
  );
}

export default App;
