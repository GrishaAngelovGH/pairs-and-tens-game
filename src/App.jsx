import Header from "components/Header"
import NumberGrid from "components/NumberGrid"

function App() {
  return (
    <div className="container-fluid">
      <div className="row vh-100 bg-primary-subtle align-items-center">
        <div className="col-md-12">
          <Header />
          <NumberGrid />
        </div>
      </div>
    </div>
  );
}

export default App;
