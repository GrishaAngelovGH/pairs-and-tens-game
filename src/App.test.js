import { render } from "@testing-library/react"

import App from "./App"

jest.mock('./components/NumberGrid/grid')

test("should render App component", () => {
  const view = render(<App />)

  expect(view).toMatchSnapshot()
})