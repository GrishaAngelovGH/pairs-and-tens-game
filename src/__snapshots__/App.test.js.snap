// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container-fluid"
      >
        <div
          class="row vh-100 bg-primary-subtle align-items-center"
        >
          <div
            class="col-md-12"
          >
            <h1
              class="text-center text-primary border border-primary shadow rounded m-2 p-2 fixed-top"
            >
              Find all pairs or sums equal to 10
            </h1>
            <div
              class="row justify-content-center"
            >
              <div
                class="col-md-6 border border-3 border-primary rounded shadow"
              >
                <div
                  class="row"
                >
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    11
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    12
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    13
                  </div>
                </div>
                <div
                  class="row"
                >
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    21
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    22
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    23
                  </div>
                </div>
                <div
                  class="row"
                >
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    31
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    32
                  </div>
                  <div
                    class="col border border-primary text-center fw-bold bg-white text-dark"
                    role="button"
                  >
                    33
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container-fluid"
    >
      <div
        class="row vh-100 bg-primary-subtle align-items-center"
      >
        <div
          class="col-md-12"
        >
          <h1
            class="text-center text-primary border border-primary shadow rounded m-2 p-2 fixed-top"
          >
            Find all pairs or sums equal to 10
          </h1>
          <div
            class="row justify-content-center"
          >
            <div
              class="col-md-6 border border-3 border-primary rounded shadow"
            >
              <div
                class="row"
              >
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  11
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  12
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  13
                </div>
              </div>
              <div
                class="row"
              >
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  21
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  22
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  23
                </div>
              </div>
              <div
                class="row"
              >
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  31
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  32
                </div>
                <div
                  class="col border border-primary text-center fw-bold bg-white text-dark"
                  role="button"
                >
                  33
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
